curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b223e41ce6b9f9a01d214.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 657b223e41ce6b9f9a01d214
+title: Task 48
+challengeType: 22
+dashedName: task-48
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: They weren't exactly big projects.
+-->
+
+# --description--
+
+You'll review the past simple negative in this task. It's formed using `was` or `were` with `not`, like `weren't` (were not) or `wasn't` (was not). It's used to talk about things that did not happen or were not true in the past.
```

Comment:
```suggestion
You'll review the past simple negative in this task. It's formed using `was` or `were` with `not`, like `weren't` (`were not`) or `wasn't` (`was not`). It's used to talk about things that did not happen or were not true in the past.
```

Maybe? Because we're referring to the actual contracted words?

Reply:
Interesting question. I slightly prefer leaving this translatable, but preventing it doesn't seem awful.

Reply:
Deferring to Nielda's example.